tests/e2e/vLLM/test_vllm.py

```diff
@@ -3,6 +3,7 @@
 import shutil
 from pathlib import Path
 
+import pandas as pd
```

Comment:
nit: Thoughts on adding to dev requirements? we should be fine for now as `datasets` installs pandas

Reply:
is dev and test the same?

Reply:
We don't have a `test` extra, only `dev` which includes all the test requirements.

Reply:
Yeah we could create a test extra.

Reply:
Adding to dev requirements for now. Will separate our test and dev requirements in a follow-up as slightly bigger change/will impact ci workflows.